docs/docsite/_themes/sphinx_rtd_theme/versions.html

```diff
@@ -33,5 +33,23 @@
 
     </div>
   </div>
+{% else %}
+  <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
+    <span class="rst-current-version" data-toggle="rst-current-version">
+      <span class="fa fa-book">Ansible Version docs</span>
```

Comment:
```suggestion
      <span class="fa fa-book">Ansible documentation version</span>
```

Reply:
OK.

Reply:
I'd like to keep 'version' in the name here... so maybe "Ansible documentation versions:" ?